release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -457,7 +457,7 @@ For more information, see the xref:../operators/operator-reference.adoc#cluster-
 [id="ocp-4-13-mco-certificate-changes"]
 ==== The MCO now syncs kubelet CA certificates on paused pools
 
-Previously, the Machine Config Operator (MCO) updated the kubelet client certificate authority (CA) certificate, `/etc/kubernetes/kubelet-ca.crt`, as a part of the regular machine config update. Starting with {product-title} (product-version}, the `kubelet-ca.crt` no longer gets updated as a part of the regular machine config update. As a result of this change, the Machine Config Daemon (MCD) automatically keeps the `kubelet-ca.crt` up to date whenever changes to the certificate occur. 
+Previously, the Machine Config Operator (MCO) updated the kubelet client certificate authority (CA) certificate, `/etc/kubernetes/kubelet-ca.crt`, as a part of the regular machine config update. Starting with {product-title} (product-version}, the `kubelet-ca.crt` no longer gets updated as a part of the regular machine config update. As a result of this change, the Machine Config Daemon (MCD) automatically keeps the `kubelet-ca.crt` up to date whenever changes to the certificate occur.
```

Comment:
header says MCO syncs the CA, but this paragraph seems to be saying it no longer does so because MCD does instead?

Reply:
(unrelated to this change i guess, but confusing regardless)

Reply:
Don't know about that one, tagging @mburke5678 who added this line

Reply:
i think the only  change needed is to switch the header from "The MCO syncs......" to "The MCD syncs..."

but yes, it would be nice to fix it.

Reply:
Gonna resolve this so it's collapsed since it's tangential to the change here and sounds like y'all have an agreement :slightly_smiling_face:

Reply:
@bparees PTAL: https://github.com/openshift/openshift-docs/pull/58103